### PR TITLE
feat(typescript): add rcbc

### PR DESCRIPTION
**Pull request by bearmit**
This pull request is related to the card: https://trello.com/c/BQvuiXmu/39-cript-scripts-rcbc

## What does this script?

The script just convert static JS objects to JSON.

The original power point graphs and the generated JSON are available here: https://drive.google.com/drive/folders/1UXFJX_NNChygw-vJOwtZY6GI90qdKGvx

## General improvements (shared with all scripts)
- The `CriptJSON `static class generate references (`"uid": "_:NAME"`) only when necessary
- We use streams instead of regular file write

### CODE REVIEW

File: scripts/typescript/package.json [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-  "description": "Set of typescript code to convert raw data (*.csv, *.ts) to CRIPT *.json format",
+  "description": "Set of typescript code to convert raw data (*.xlsx, *.csv, *.ts) to CRIPT *.json format",
   "main": "index.js",
   "scripts": {
-    "afrl": "npx ts-node ./src/afrl/index.ts",
-    "rcbc": "npx ts-node ./src/rcbc/index.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "npx tsc",
+    "afrl": "npx ts-node ./src/afrl",
+    "bcdb": "npx ts-node ./src/bcdb",
```
Comment (bearmit) [inline scripts/typescript/package.json:9]: I was working on two datasets at the time, that's why you can see some traces of it. Another PR is coming.

**Comment by nh916** [comment]
I don't know enough to provide an effective review, but I ran this PR locally and it seems to work fine. 

The only note I have is regarding `Link to TypeScript source code` and I think that could be improved and changed to instead say `TypeScript source code on GitHub`. I think that could also earn us some SEO points as well. 
I wrote a small [SEO links wiki](https://github.com/C-Accel-CRIPT/criptscripts/wiki/How-to-use-links-effectively) within the [CRIPT Scripts repository](https://github.com/C-Accel-CRIPT/criptscripts) that I thought could be helpful for us

I leave it for you to merge this PR in whenever you think it is good, and then please [create a release](https://github.com/C-Accel-CRIPT/criptscripts/releases), generate release notes, and let the CRIPT channel know. Or just let me know when you merge it in and I can do all of that!
These steps should eventually be automated if we decide to stick with this repository for the long term

**Review by nh916 — APPROVED** [review]

**Comment by bearmit** [comment]
@nh916 Thanks. I would like to wait a little bit in case @shijiale0609 have a moment to look at the graph sources (in [src/rcbc/data/graph](https://github.com/C-Accel-CRIPT/criptscripts/pull/12/files#diff-93d066bdcb56fd6894a6bb502f3983db66c588411546c517d86fead896604f0e)) next week.

**Review by shijiale0609 — CHANGES_REQUESTED** [review]
computations.ts, PPV-b-PI-89. There is no computation in the graph which was polished by Brad. see the last slide https://docs.google.com/presentation/d/1eXM7870YM2sxQQvMFahI2Ox8-EFCGaw_/edit#slide=id.p3

**Comment by nh916** [comment]
@bearmit be sure to delete branches that you are not using, its just good to keep the repository clean

**Pull request merged**